agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -69,6 +69,42 @@ static bwl::WiFiSec platform_to_bwl_security(const std::string &sec)
     }
 }
 
+static std::string wsc_to_bwl_authentication(WSC::eWscAuth authtype)
+{
+    switch (authtype) {
+    case WSC::eWscAuth::WSC_AUTH_OPEN:
+        return "NONE";
+    case WSC::eWscAuth::WSC_AUTH_WPAPSK:
+        return "WPA-PSK";
+    case WSC::eWscAuth::WSC_AUTH_SHARED:
+        return "SHARED";
+    case WSC::eWscAuth::WSC_AUTH_WPA:
+        return "WPA";
+    case WSC::eWscAuth::WSC_AUTH_WPA2:
+        return "WPA2";
+    case WSC::eWscAuth::WSC_AUTH_WPA2PSK:
+        return "WPA2-PSK";
+    default:
+        return "INVALID";
+    }
+}
+
+static std::string wsc_to_bwl_encryption(WSC::eWscEncr enctype)
+{
+    switch (enctype) {
+    case WSC::eWscEncr::WSC_ENCR_NONE:
+        return "NONE";
+    case WSC::eWscEncr::WSC_ENCR_WEP:
+        return "WEP";
+    case WSC::eWscEncr::WSC_ENCR_TKIP:
+        return "TKIP";
+    case WSC::eWscEncr::WSC_ENCR_AES:
+        return "AES";
+    default:
+        return "INVALID";
+    }
+}
```

Comment:
I think it will better be on son_wireless_utils.

Reply:
Moved.